metadata.json

```diff
@@ -36,6 +36,12 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "FreeBSD"
```

Comment:
freebsd isnt rolling release (well, kind of), so we shul list the versions we support. same for openbsd

Reply:
Yeah, I wasn't sure which versions this would work with, and was mostly relying on examples from other modules. Because of the ports system, my guess is that the range of present and future supported versions might be pretty broad, though I don't know how far back we'd want to support.

I do use FreeBSD and OpenBSD (FBSD since the 4.x days), but not with Puppet, so I didn't really know what values are reasonable here. Suggestions or PR to my PR are welcome 😉